src/mygrad/math/arithmetic/ops.py

```diff
@@ -40,6 +47,35 @@ def backward_var(self, grad, index, **kwargs):
         return grad
 
 
+class IAdd(Add):
+    def __call__(self, inplace_target, other) -> np.ndarray:
+        """ Performs a += b
+
+        Parameters
+        ----------
+        inplace_target : mygrad.Tensor
+        other : mygrad.Tensor
+
+        Returns
+        -------
+        inplace_target_data : numpy.ndarray
+
+        Notes
+        -----
+        Note that inplace_target will be replaced with its un-mutated
+        value by ``Tensor._inplace_op`` prior to backprop. Thus we need
+        not worry about caching inplace_targer and rewriting the backprop
```

Comment:
My only contribution is that you spelled this `inplace_targer` all over the place